shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/admin/mysql/executor/ShowVersionExecutor.java

```diff
@@ -39,15 +43,31 @@ public final class ShowVersionExecutor implements DatabaseAdminQueryExecutor {
     
     public static final String FUNCTION_NAME = "version()";
     
+    private SelectStatement sqlStatement;
```

Comment:
Could we make it final and use `@RequiredArgsConstructor`.

Reply:
done